### ISSUE TITLE
Crashes and Double question

### ISSUE THREAD

**Issue by Mannshoch** [westnordost/StreetComplete#1001]
I did a little hiking and tried to use Streetcomplete on It. At first I had the Problem that Streetcomplet dont asked me any Question beside missing streetnames and in some rare situation the surface. After I moved the Question for Streetnames from top priority position to the lowest I got a huge amount of Questions.
But now I had the Problem, that if I answered some surface question I got double asked about speedlimit. And thens sometimes Streetcomplite crasehed.

Below I add two crash-log.

```
Device: Fairphone FP2, Android 6.0.1
Thread: main
App version: 4.1
Stack trace:
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
at android.view.LayoutInflater.from(LayoutInflater.java:228)
at android.view.ContextThemeWrapper.getSystemService(ContextThemeWrapper.java:119)
at android.support.v7.app.AlertController$AlertParams.<init>(AlertController.java:903)
at android.support.v7.app.AlertDialog$Builder.<init>(AlertDialog.java:325)
at de.westnordost.streetcomplete.view.dialogs.AlertDialogBuilder.<init>(AlertDialogBuilder.java:14)
at de.westnordost.streetcomplete.quests.max_speed.AddMaxSpeedForm.askUrbanOrRural(AddMaxSpeedForm.java:238)
at de.westnordost.streetcomplete.quests.max_speed.AddMaxSpeedForm.determineImplicitMaxspeedType(AddMaxSpeedForm.java:193)
at de.westnordost.streetcomplete.quests.max_speed.AddMaxSpeedForm.bridge$lambda$1$AddMaxSpeedForm(AddMaxSpeedForm.java)
at de.westnordost.streetcomplete.quests.max_speed.AddMaxSpeedForm$$Lambda$16.run(Unknown Source)
at de.westnordost.streetcomplete.quests.max_speed.AddMaxSpeedForm.lambda$confirmNoSignSlowZone$8$AddMaxSpeedForm(AddMaxSpeedForm.java:231)
at de.westnordost.streetcomplete.quests.max_speed.AddMaxSpeedForm$$Lambda$6.onClick(Unknown Source)
at android.support.v7.app.AlertController$ButtonHandler.handleMessage(AlertController.java:162)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5417)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:107)
```

```
Device: Fairphone FP2, Android 6.0.1
Thread: main
App version: 4.1
Stack trace:
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
at android.view.LayoutInflater.from(LayoutInflater.java:228)
at de.westnordost.streetcomplete.quests.max_speed.AddMaxSpeedForm.confirmNoSignSlowZone(AddMaxSpeedForm.java:218)
at de.westnordost.streetcomplete.quests.max_speed.AddMaxSpeedForm.lambda$addOtherAnswers$3$AddMaxSpeedForm(AddMaxSpeedForm.java:148)
at de.westnordost.streetcomplete.quests.max_speed.AddMaxSpeedForm$$Lambda$2.run(Unknown Source)
at de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment.lambda$null$1$AbstractQuestAnswerFragment(AbstractQuestAnswerFragment.java:124)
at de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment$$Lambda$3.onMenuItemClick(Unknown Source)
at android.widget.PopupMenu.onMenuItemSelected(PopupMenu.java:252)
at com.android.internal.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:761)
at com.android.internal.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:152)
at com.android.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:904)
at com.android.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:894)
at com.android.internal.view.menu.MenuPopupHelper.onItemClick(MenuPopupHelper.java:200)
at android.widget.AdapterView.performItemClick(AdapterView.java:310)
at android.widget.AbsListView.performItemClick(AbsListView.java:1145)
at android.widget.AbsListView$PerformClick.run(AbsListView.java:3066)
at android.widget.AbsListView$3.run(AbsListView.java:3903)
at android.os.Handler.handleCallback(Handler.java:739)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5417)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:107)
```

**Comment by rugk** [comment]
The "double opening" of a quest has been reported in https://github.com/westnordost/StreetComplete/issues/973 and is fixed in the next release.

The reason why SC shows you only a few quests when started [is explained in the FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#Why_does_StreetComplete_show_so_few_quests_when_I_first_start_the_app.3F).

As for the crash report, I think someone else has to have a look on it, so that's what we can handle in this issue. :smile:

**Comment by westnordost** [comment]
So is this fixed in v5.0-beta1?

**Comment by Mannshoch** [comment]
Seems resolved.
